harness/determined/cli/config_policies.py

```diff
@@ -63,21 +104,29 @@ def delete_config_policies(args: argparse.Namespace) -> None:
         "manage config policies",
         [
             cli.Cmd(
-                "describe",
-                describe_config_policies,
-                "describe config policies",
+                "get",
+                get_config_policies,
+                "get config policies",
```

Comment:
What was wrong with "describe"?

Reply:
I want to use what's already familiar / introduced to the CLI. Do you know if we have other commands that use describe or set?

Reply:
I see there's some "list".

Reply:
we have `det w bindings set ...` and ur right, `describe` is more commonly used in our CLI.
I'll change this back!